.eslintrc.cjs

```diff
@@ -1,9 +1,28 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
   root: true,
-  extends: ['@documenso/eslint-config'],
+  extends: [
+    '@documenso/eslint-config',
+    'plugin:package-json/recommended',
+    'plugin:import/recommended',
+  ],
+  plugins: ['package-json', 'import', 'simple-import-sort'],
   rules: {
     '@next/next/no-img-element': 'off',
+    'no-multi-spaces': [
+      'error',
+      {
+        ignoreEOLComments: false,
+        exceptions: {
+          BinaryExpression: false,
+          VariableDeclarator: false,
+          ImportDeclaration: false,
+          Property: false,
+        },
+      },
+    ],
+    'import/named': 'off',
+    'no-duplicate-imports': 'error',
```

Comment:
What's up with these two stragglers?

Reply:
1. It is to verify if all the named imports are part of the set of named exports in the referenced module. More Info: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/named.md
2. This is to prevent duplicate imports, there was one under 'create.ts` file for formidable package